backend/infrahub/core/diff/query/merge.py

```diff
@@ -390,7 +390,7 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: Any) -> None:
             WHERE type(r_prop) = prop_type
             AND r_prop.status = prop_rel_status
             AND r_prop.from <= $at
-            AND (r_prop.to >= $at OR r_prop.to IS NULL)
+            AND (r_prop.to > $at OR r_prop.to IS NULL)
```

Comment:
this off-by-one time error was causing a bug where the merge would delete an edge that it had just updated because the `to` time was set to $at